src/core/modules/ZodiacalLight.cpp

```diff
@@ -250,7 +250,8 @@ void ZodiacalLight::draw(StelCore* core)
 
 			float oneMag=0.0f;
 			extinction.forward(vertAltAz, &oneMag);
-			float extinctionFactor=std::pow(0.4f , oneMag)/bortleIntensity; // drop of one magnitude: factor 2.5 or 40%, and further reduced by light pollution
+			float extinctionFactor=std::pow(0.4f , oneMag) * (1.1f-bortleIntensity*0.1f);// Drop of one magnitude: factor 2.5 or 40%, and further reduced by light pollution.
+			//float extinctionFactor=std::pow(0.4f , oneMag)/bortleIntensity; // Atque 2021-02-22: This makes the ZL invisible with the slightest LP.
```

Comment:
please remove the commented old line. The commit message will include your name/date/comment.